### PR TITLE
fix update version for create; add type linting to docs and create

### PR DESCRIPTION
**Pull request by daKmoR**

### CODE REVIEW

File: docs/developing/types.md [hunk 1]
```diff
@@ -18,7 +18,18 @@ Types provide valuable information about the nature of code, and help identify t
 ## Open Web Components Types
 
 For most of our products we do offer types via JSDocs.
-In order to utalize them you will need to add something to your setup.
+In order to utalize them you will need to set them up for your project.
```
Comment (thepassle) [inline docs/developing/types.md]:
```suggestion
In order to utilize types, you will need to set them up for your project.
```

File: docs/developing/types.md [hunk 2]
```diff
@@ -18,7 +18,18 @@ Types provide valuable information about the nature of code, and help identify t
...
+
+### Why should I care?
```
Comment (thepassle) [inline docs/developing/types.md]:
```suggestion
### Why types?
```

File: docs/developing/types.md [hunk 3]
```diff
@@ -18,7 +18,18 @@ Types provide valuable information about the nature of code, and help identify t
...
+
+Simply speaking this will improve your developer experience by giving you
+
+- Nice intellisense for @open-wc tools/helpers
+- Direct go to the source of @open-wc code via F12 (in vs code)
```
Comment (thepassle) [inline docs/developing/types.md]:
```suggestion
- Ability to jump directly to the source of @open-wc code via F12 (in vs code)
```

File: docs/developing/types.md [hunk 4]
```diff
@@ -18,7 +18,18 @@ Types provide valuable information about the nature of code, and help identify t
...
 
 For most of our products we do offer types via JSDocs.
-In order to utalize them you will need to add something to your setup.
+In order to utalize them you will need to set them up for your project.
+
+### Why should I care?
+
+Simply speaking this will improve your developer experience by giving you
+
+- Nice intellisense for @open-wc tools/helpers
```
Comment (thepassle) [inline docs/developing/types.md]:
```suggestion
- Awesome intellisense for @open-wc tools/helpers
```

File: docs/linting/linting-types.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# Linting Types
+
+Use [TypeScript](https://www.typescriptlang.org/) to type lint your es6 JSDoc annotaded code.
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
Use [TypeScript](https://www.typescriptlang.org/) to type lint your es6 JSDoc annotated code.
```

File: docs/linting/linting-types.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Add these scripts to your package.json
+  ```js
+  "scripts": {
+    "lint:types": "tsc",
+  },
+  ```
+
+## What you get
+
+For a general information about types please see [develping/types](../developing/types).
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
For general information about types please see [develping/types](../developing/types).
```

File: docs/linting/linting-types.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+- Nice intellisense for @open-wc tools/helpers
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
- Awesome intellisense for @open-wc tools/helpers
```

File: docs/linting/linting-types.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+- Direct go to the source of @open-wc code via F12 (in vs code)
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
- Ability to jump directly to the source of @open-wc code via F12 (in vs code)
```

File: docs/linting/linting-types.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+- Types Safty => why that is a good thing see [develping/types](../developing/types).
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
- Type safety (To read more about why type safety can help you, go to: [develping/types](../developing/types)).
```

File: docs/linting/linting-types.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+## Usage
+
+Run:
+- `npm run lint:types` to check if you have type type errors
```
Comment (thepassle) [inline docs/linting/linting-types.md]:
```suggestion
- `npm run lint:types` to check if you have type errors
```

**Comment by daKmoR** [comment]
I'm pretty sure I said that before... but I just love the suggestion feature of github 😍 

and thxxxxx @thepassle for all the suggestions 🤗

**Comment by thepassle** [comment]
Me too, it's so nice. Really miss that at work 🙈

File: docs/developing/types.md [hunk 11]
```diff
@@ -18,7 +18,18 @@ Types provide valuable information about the nature of code, and help identify t
 ## Open Web Components Types
 
 For most of our products we do offer types via JSDocs.
-In order to utalize them you will need to add something to your setup.
+In order to utilize types, you will need to set them up for your project.
+
+### Why types?
+
+Simply speaking this will improve your developer experience by giving you
```
Comment (thepassle) [inline docs/developing/types.md]:
```suggestion
Types can help improve your developer experience by giving you:
```

File: docs/linting/linting-types.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+Run:
+- `npm run lint:types` to check if you have type errors
+
+## Type linting error example
+
+```bash
+$ npm run lint:types
+
+> foo-bar@ lint:types /path/to/foo-bar
```
Comment (thepassle) [inline docs/linting/linting-types.md]: is this line correct?

Comment (daKmoR) [reply docs/linting/linting-types.md]: jup that's how npm tells what it's invoking where...

is it confusing - and not helpful for an example? then we could remove it

**Pull request merged**